### ISSUE TITLE
[BUG] Translating a post doesn't work

### ISSUE THREAD

**Issue by roydbt** [mastodon/mastodon-ios#1064]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When trying to translate a post, I get this error:

> **Note**
> Translation failed. Maybe the administrator has not enabled translations on this server or this server is running an older version of Mastodon where translations are not yet supported.

I am getting this error for every post I'm trying to translate, doesn't matter if it's in English or not, and the button is always labeled as "Translate from English".

### Expected Behavior

I expect this button to work.

### Steps To Reproduce

1. See a post you would like to translate
2. Tap '...'
3. Press "Translate from English"
4. Voilà!

### Environment

```markdown
- Device: iPhone 13 Pro
- OS: iOS 16.5
- Version: v2023.9
- Build: 324
```

### Anything else?

My account is on mastodon.social and the translation worked a few days ago.

**Comment by zeitschlag** [comment]
Hello @roydbt,

I just tried to reproduce your issue and it worked as expected: Posts in English were successfully translated to German (the language defined in my Mastodon-account-settings). But translating a German post to German resulted in that error (and an HTTP `403`):

<details>

![](https://github.com/mastodon/mastodon-ios/assets/2580019/53018de0-99f9-46d9-b724-ac3189e2c56c)

</details>

In that case we could think about showing a more meaningful message or not providing the translate-option. Assuming your languate isn't set to German, can you translate this post:

https://chaos.social/@zeitschlag/110949059473167329

?

**Comment by zeitschlag** [comment]
Closed in #1135
